### DISCUSSION TITLE
What is the difference between these?

### DISCUSSION THREAD

**Discussion by gregeeh** [rakshasa/rtorrent#1149] [Q&A] [answered]
Hi,

I've noticed various people use different values to set the default download directory.  I've seen the following:

directory.default
directory.default.set
directory

Is there a difference between each of these?

TIA

**Comment by kannibalox** [comment, marked as answer]
`directory.default` returns the current default directory, while `directory.default.set` sets it. `directory` is just an alias for `directory.default.set` and should be avoided.

**Reply by gregeeh** [reply]
Thank you very much.  Searched Google high and low and could not find this answer.  Thanks again.
